packages/rn-tester/js/examples/KeyViewLoopExample/KeyViewLoopExample.js

```diff
@@ -111,6 +111,30 @@ class FocusTrapExample extends React.Component<{}> {
   }
 }
 
+class FocusRingExample extends React.Component<{}> {
+  render() {
+    return (
+      <View>
+        <View style={styles.keyView} focusable={true} enableFocusRing={true}>
+          <Text>Enabled</Text>
+        </View>
+        <View style={styles.keyView} focusable={true} enableFocusRing={false}>
+          <Text>Disabled</Text>
+        </View>
+        <View style={styles.keyView} focusable={true}>
+          <Text>Default</Text>
+        </View>
+        <View style={styles.keyView} focusable={false}>
+          <Text>Not focusable</Text>
+        </View>
+        <View style={styles.keyView} focusable={true} enableFocusRing={true}>
+          <Text>Enabled</Text>
+        </View>
```

Comment:
I feel like this should be in a separate page from KeyViewLoopExample, if it's not testing keyViewloop behavior (i.e: you aren't setting that prop).

Reply:
Done